fury/ui/tests/test_containers.py

```diff
@@ -250,113 +250,118 @@ def test_ui_image_container_2d(interactive=False):
 
 
 def test_ui_tab_ui(interactive=False):
-    filename = 'test_ui_tab_ui'
-    recording_filename = pjoin(DATA_DIR, filename + '.log.gz')
-    expected_events_counts_filename = pjoin(DATA_DIR, filename + '.json')
+    positions = ['top', 'bottom']
+    for i in positions:
```

Comment:
I doubt using for loop may cause few issues moving forward. It would make it difficult to understand in which loop did the test failed whether it was the `top` case or `bottom` case.
Please have a look at other tests where we have checked multiple parameters in a single test using multiple instance of same UI element.

Reply:
Got it I will try doing it the other way
Thank you

Reply:
As far as I remember we have kept the same instance of show manager and other things like event counter and elements but just created the UI object with different parameters for testing. Let me see your previous test and will comment on it.